manifests/git_repo.pp

```diff
@@ -28,4 +28,12 @@
     user     => $katello_devel::user,
   }
 
+  file { "${katello_devel::deployment_dir}/${title}/.git/hooks/commit-msg":
+    ensure  => file,
+    owner   => $::katello_devel::user,
+    group   => $::katello_devel::group,
+    mode    => '0755',
+    content => template('katello_devel/commit-msg.rb.erb'),
+    require => File["${katello_devel::deployment_dir}/${title}"],
```

Comment:
Ah that should have been `Vcsrepo` not `File`. I don't think autorequires works with that resource type? 

I'll push it when https://github.com/Katello/puppet-katello_devel/pull/107 is merged. Tests will fail without it.